tests/python/contrib/test_arm_compute_lib/infrastructure.py

```diff
@@ -25,15 +26,22 @@
 from tvm.contrib import graph_runtime
 from tvm.relay.op.contrib import arm_compute_lib
 from tvm.contrib import util
+from tvm.autotvm.measure import request_remote
 
 
 class Device:
-    """Adjust the following settings to connect to and use a remote device for tests."""
-    use_remote = False
-    target = "llvm -mtriple=aarch64-linux-gnu -mattr=+neon"
-    # Enable cross compilation when connecting a remote device from a non-arm platform.
-    cross_compile = None
-    # cross_compile = "aarch64-linux-gnu-g++"
+    """
+    Use the following environment variables to connect to and use a remote device.
+    The RPC mechanism can be used in the case of compiling an ACL module on an x86 machine
+    but running on an AArch64 machine.
+    """
+    rpc_host = os.environ.get("TVM_ARM_COMPUTE_LIB_RPC_HOST", "localhost")
+    rpc_port = int(os.environ.get("TVM_ARM_COMPUTE_LIB_RPC_PORT", 9090))
+    rpc_device_key = os.environ.get("TVM_ARM_COMPUTE_LIB_RPC_DEVICE_KEY", "")
+    target = os.environ.get("TVM_ARM_COMPUTE_LIB_TARGET", "llvm -mtriple=aarch64-linux-gnu -mattr=+neon")
```

Comment:
I wonder if we can make these generic environment variables provided in a common manner in the test infrastructure TVM_RPC_HOST, TVM_RPC_PORT and TVM_RPC_DEVICE_KEY. There's nothing ACL specific about this ... 

I am not sure if there are such provisions in pytest for such extensions which would be pretty cool.

Reply:
I did have a think about this, although different tests may require different machines. For example CoreML uses similar environment variables to connect to a machine running ios. I feel like this approach may enforce that we use the same device for every test which uses RPC?

Reply:
Yeah we'd need ways of skipping the test on devices that aren't supported.